README.md

```diff
@@ -1,3 +1,4 @@
+[![Stories in Ready](https://badge.waffle.io/mesosphere/marathon.png?label=ready,gui&title=Ready)](https://waffle.io/mesosphere/marathon?label=gui)
```

Comment:
Yes, but I would not understand that as an external user/reader of the repository. I still don't see what value this provides, the number, I guess, is not important to me, rather which ones. I also don't think they should do it like this in Marathon. I would much rather just have the link and a description of what the user could expect to see.